var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -22,30 +22,27 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+from spack.util.prefix import Prefix
 from spack import *
 import os
-import re
 
-from spack.pkg.builtin.intel import IntelInstaller, filter_pick, \
-    get_all_components
 
-
-class IntelParallelStudio(IntelInstaller):
+class IntelParallelStudio(IntelPackage):
     """Intel Parallel Studio."""
 
     homepage = "https://software.intel.com/en-us/intel-parallel-studio-xe"
 
-    version('professional.2017.4', '669b45b8dc9826ba018966367b802f98',
-            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11481/parallel_studio_xe_2017_update4.tgz')
-    version('cluster.2017.4', '669b45b8dc9826ba018966367b802f98',
-            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11481/parallel_studio_xe_2017_update4.tgz')
-    version('composer.2017.4', '3cd2a5763bf2ebe97889a067a1a7c800',
-            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11485/parallel_studio_xe_2017_update4_composer_edition.tgz')
+    version('professional.2017.4', '27398416078e1e4005afced3e9a6df7e',
+            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11537/parallel_studio_xe_2017_update4.tgz')
+    version('cluster.2017.4',      '27398416078e1e4005afced3e9a6df7e',
+            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11537/parallel_studio_xe_2017_update4.tgz')
+    version('composer.2017.4',     'd03d351809e182c481dc65e07376d9a2',
+            url='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11541/parallel_studio_xe_2017_update4_composer_edition.tgz')
```

Comment:
I think Intel keeps updating the build number and re-releasing the latest version.